Add a section on Slack Admins (#1273)
JuliaLang/www.julialang.org: slack.md

# The Julia Language Slack

The Julia Language has an active Slack workspace/community with over 8,000 members. Slack is a space to have quick and informal correspondence with others in the community. We ask that if you have Julia usage questions, you post them on [Discourse](https://discourse.julialang.org) so others can benefit from them. You could also use Stack Overflow but the Discourse community is much more welcoming (in our opinion).

## You can find the link to join below:
[Join us in the Julia Language Slack](https://join.slack.com/t/julialang/shared_invite/zt-nmal0i0x-LcYEtdnTameGsXmBzMzgog)

### Issues using the link above? 
[Send us a note so we can fix that!](mailto:logan@julialang.org)

### Admins
The [Julia Community Standards](https://julialang.org/community/standards/) are enforced on Slack by our Slack Admins. 

> **If you see or experience any violation of those standards, or feel unsafe or upset, please contact us by sending an email to [slackadmins@julialang.org](mailto:slackadmins@julialang.org).** All communications will be treated confidentially but incidents may be shared the Julia Stewards.

Reviewer: ```suggestion
**If you see or experience any violation of those standards, or feel unsafe or upset, please contact us by sending an email to [slackadmins@julialang.org](mailto:slackadmins@julialang.org).** All communications will be treated confidentially but incidents may be shared the Julia Stewards (who will treat all information confidentially as well).
```

You can also contact the [Julia Stewards](https://julialang.org/community/stewards/) by emailing [stewards@julialang.org](mailto:stewards@julialang.org), in particular if your concern or conflict involves the admins and/or you would prefer a third party to resolve it.

> Slack lacks adequate moderations tools so at the present moment, you will have to ping one of the following moderators directly if you think the Julia Community Standards have been violated or otherwise need the support of an Admin. You can find a list of our current Admins here (Slack sign-in required): [https://julialang.slack.com/account/workspace-settings#admins](https://julialang.slack.com/account/workspace-settings#admins)

Reviewer 2: Rephrasing so that there isn't a URL visible, to improve accessibility:
```suggestion
Slack lacks adequate moderations tools so at the present moment, you will have to ping one of the following moderators directly if you think the Julia Community Standards have been violated or otherwise need the support of an Admin. See [the list of our current Slack Admins](https://julialang.slack.com/account/workspace-settings#admins) (Slack sign-in required)
```
